### PR TITLE
Add brave and opera to browser appnames

### PR DESCRIPTION
**Pull request by johan-bjareholt**
https://forum.activitywatch.net/t/web-watcher-doesnt-work-with-brave

Does not work without https://github.com/ActivityWatch/aw-watcher-web/pull/37

### CODE REVIEW

**Comment by ErikBjare** [comment]
There's just one thing that worries me, there should be one bucket for each of Chrome, Chrome Beta, Firefox, Firefox Nightly, Brave, Opera, etc to prevent issues where there are two instances concurrently (still achievable by using profiles or portable installs, but far less likely).

**Comment by johan-bjareholt** [comment]
@ErikBjare Yeah, I wrote about that in the forum thread. We currently just check the useragent for firefox/chrome but that should be possible to extend.

File: src/queries.js [hunk 1]
```diff
@@ -44,9 +44,18 @@ export function appQuery(appbucket, limit) {
 export function browserSummaryQuery(browserbucket, windowbucket, afkbucket, count, filterAFK) {
   var browser_appnames = "";
   if (browserbucket.endsWith("-chrome")){
-    browser_appnames = JSON.stringify(["Google-chrome", "chrome.exe", "Chromium", "Google Chrome", "Chromium-browser", "Chromium-browser-chromium", "Google-chrome-beta", "Google-chrome-unstable"]);
+    browser_appnames = JSON.stringify([
+        "Google-chrome", "chrome.exe", "Google Chrome",
+        "Chromium", "Chromium-browser", "Chromium-browser-chromium",
+        "Google-chrome-beta", "Google-chrome-unstable",
+        // FIXME: Are these correct? Does the bucketname of opera and brave end with "-chrome"?
+        "opera.exe", "brave.exe",
```
Comment (krmbzds) [inline src/queries.js]: For Brave on macOS it is `Brave Browser`.

```
kerem             5202   0.0  1.9  5354244 315044   ??  S     5:13PM   2:38.60 /Users/kerem/Applications/Brave Browser.app/Contents/MacOS/Brave Browser
```

Comment (krmbzds) [reply src/queries.js]: For Vivaldi on macOS it's just `Vivaldi`. I downloaded Vivaldi browser just to check this out.

```
kerem            17897   0.0  0.9  4883304 145412   ??  S     5:51PM   0:06.62 /Users/kerem/Applications/Vivaldi.app/Contents/MacOS/Vivaldi -psn_0_8038314
```

Comment (krmbzds) [reply src/queries.js]: For Opera on macOS it is `Opera`.

```
kerem            22320   0.0  1.0  5345560 162692   ??  S     6:01PM   0:03.83 /Users/kerem/Applications/Opera.app/Contents/MacOS/Opera -psn_0_9988486
```

**Comment by johan-bjareholt** [comment]
@ErikBjare Updated, now actually works. Tested with Opera with the following aw-watcher-web patch
https://github.com/ActivityWatch/aw-watcher-web/pull/37

Together with the changes to choose all buckets in #141 by @nicolae-stroncea it works very well with running firefox, chrome and opera all with watchers running and visualizing them summaried together.

**Comment by ErikBjare** [comment]
@johan-bjareholt Didn't we talk about `aw-watcher-web-chrome` becoming `aw-watcher-web-chromium` for Chromium? If so shouldn't we split the `appnames['chrome']` into `chrome` and `chromium` variants?

**Comment by johan-bjareholt** [comment]
@ErikBjare That is impossible to know, this is the useragent for Chromium on Linux

> Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/75.0.3770.100 Safari/537.36

No mention at all that it's Chromium.

**Pull request merged**